src/main.ts

```
@@ -452,36 +483,69 @@ async function generateDetailedReviews(
+
+                reviews.push({
+                    path: fileIssue.path,
+                    lineNumber: issue.lineNumber,
+                    severity: normalizeSeverity(response.value.severity ?? 'info'),
+                    analysis: response.value.analysis ?? 'No analysis provided',
+                    suggestion: response.value.suggestion ?? '',
+                })
+            } catch (error) {
+                logger.error(`Error processing review for ${fileIssue.path}:${issue.lineNumber}:`, error)
                 continue
             }
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)